volby/index.html

```diff
@@ -118,7 +104,19 @@ <h3 class="o-section__heading t-h4-super">Pardubice VI</h3>
   </div>
 
   <hr>
+  <div class="pce-hero">
+      <div class="pce-hero__content">
+        <img alt="Pirátští poslanci" src="https://i.imgur.com/UYUhLsl.png" />
```

Comment:
@dominikbecka Pls, všechny fotky tohodle typu držme v rámci repository, ne na externí službě.

Reply:
To už bylo v minulým omylem nechaný, chtěl jsem to opravit a zapomněl na to. Dotáhnu.